javascript/javascript1/week4/js1w4-hw.js

```diff
@@ -0,0 +1,138 @@
+// VOICE ASSISTANT COMMAND HOME WORK
+
+let string = "Hello my name is Benjamin";
+let stringArray = string.split(" ");
+let name = stringArray[stringArray.length - 1];
+let names = [];
+let whoAmI = "What is my name";
+let day = "What day is it today";
+let todo = [];
+let myTodoList = "What is on my todo";
+let favDish = [];
+let event = [];
+let eventQuestion = "What am I doing this week";
+
+function getReply(command) {
```

Comment:
If I do `console.log(getReply("Hello my name is Florin"));`  then your function would not work correctly. Can you spot the problem ? 
 You should first think how can you retrieve name `Florin` or `Benjamin`. Then add it to your array of `names`